commands/hugo.go

```diff
@@ -578,9 +580,16 @@ func (c *commandeer) copyStatic() error {
 	syncer.DestFs = c.Fs.Destination
 	// Now that we are using a unionFs for the static directories
 	// We can effectively clean the publishDir on initial sync
-	syncer.Delete = c.Cfg.GetBool("cleanDestinationDir")
+	syncer.Delete = c.Cfg.GetBool("cleanDestinationDir") || c.Cfg.GetBool("cleanDestinationDirKeepDotFolders")
```

Comment:
We have enough flag. 

I would say that having a cleanDestinationDir command that also deleted hidden folders ("." files) was a surprise for many. So I suggest we just make this the new default and make sure to document it. People who want to remove the entire folder can use `rm -rf`.